signers/signer-evm/src/signer.ts

```diff
@@ -15,10 +17,8 @@ import {
 import { cleanEvmError, getTenderlyError, waitMs } from './helper.js';
```

Comment:
Please update cleanEvmError code based on this document:
https://docs.ethers.org/v6/api/utils/errors/

Reply:
I checked it, shortMessage contains a suitable error messages in most cases except "UNKNOWN_ERROR" which is handled by checking the message of the inner error object.

Reply:
You could also update catch code in wait in evm signer like this:

```
    } catch (error) {
      if (isError(error, "TRANSACTION_REPLACED")) {
        const reason = error.reason
        if (reason === "cancelled") {
          throw new SignerError(
            SignerErrorCode.SEND_TX_ERROR,
            undefined,
            'Transaction replaced and canceled by user',
            undefined,
            error
          );
        }
        return { hash: error.replacement.hash, response: error.replacement };
      } else if (isError(error, "CALL_EXCEPTION")) {
        const tError = await getTenderlyError(chainId, txHash);
```

Reply:
Can you give me an example of reproducing provider error?

Reply:
I couldn't reproduce such error, I said that based on the existing code. So I removed the existing code and used ethers for error handling. I think `rango-types` package should be updated as well. Also updated `wait` function as you said.
[fix: improve errors](https://github.com/rango-exchange/rango-client/pull/842/commits/dccb0673a12cc9a64787230e5bdbb42ec2c68a6f)